packages/shared/src/styles/base.css

```diff
@@ -166,6 +167,7 @@ body {
   --theme-background-bun: theme('colors.bun.20');
   --theme-background-onion: theme('colors.onion.20');
   --theme-background-pepper: theme('colors.pepper.20');
+  --theme-background-cabbage-opacity-24: theme('colors.cabbage.50')24;
```

Comment:
Not sure if I am looking at the right Sketch part relevant to the usage of this.

The spec says 40 and opacity of 18%.

`18%` opacity I think should be `2B`.

Link for reference: https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

![Screenshot 2023-05-04 at 8 22 37 PM](https://user-images.githubusercontent.com/13744167/236202898-c40b79f5-d232-4ad7-871b-5de22c1a97dc.png)

Reply:
Checked with Tomer, it should be `24%` to match rest of the elements.